bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/MSSMD/MSSMD_proc_card.dat

```diff
@@ -0,0 +1,19 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
btw, these lines are not needed since they will be imported by default

Reply:
Hi, I've updated the file.